### PR TITLE
Document how to debug the Hexatomic product from Eclipse.

### LINKED ISSUE #27: Document how to debug/start the application from Eclipse

### PR DESCRIPTION
**Pull request by thomaskrause**
This will close #27.

### CODE REVIEW

File: docs/dev/src/development/debug-with-eclipse.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Debug with Eclipse
+
+Hexatomic is a modular project and while it is possible to debug a user-defined sub-set of the bundles,
+in most cases debugging the complete application is more useful.
+
+To debug or start Hexatomic with all the features and bundles, you have to open the  `org.corpus_tools.hexatomic.product` project which contains the Hexatomic product definition.
+In this project open the `org.corpus_tools.hexatomic.product` file in Eclipse.
```
Comment (sdruskat) [inline docs/dev/src/development/debug-with-eclipse.md]:
```suggestion
In this project, open the `org.corpus_tools.hexatomic.product` file in Eclipse.
```

;)

File: docs/dev/src/development/debug-with-eclipse.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+![Location of the product definition file in the Project Explorer](./product-file-location.png)
+
+To debug Hexatomic, click on the `Launch an Eclipse application in Debug mode` link on the overview tab of the product configuration file editor.
```
Comment (sdruskat) [inline docs/dev/src/development/debug-with-eclipse.md]:
```suggestion
To debug Hexatomic, click on the `Launch an Eclipse application in Debug mode` link in the overview tab of the Product Configuration Editor.
```

File: docs/dev/src/development/debug-with-eclipse.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+To just run the product, but not debug it, choose `Launch an Eclipse application` on the same page.
```
Comment (sdruskat) [inline docs/dev/src/development/debug-with-eclipse.md]:
```suggestion
To run the product without debugging, click `Launch an Eclipse application` on the same page.
If you don't see the Product Configuration Editor, you may have opened the file with a different editor instead.
In this case, right-click on the `org.corpus_tools.hexatomic.product` file, and choose *Open With* > *Product Configuration Editor*.
```

File: docs/dev/src/SUMMARY.md [hunk 4]
```diff
@@ -4,6 +4,7 @@
 - [Development](./development/README.md)
     - [Development setup](./development/setup.md)
         - [Getting the source code](./development/getting-the-source-code.md)
+        - [Debug with Eclipse](./development/debug-with-eclipse.md)
```
[the remark this answers is not in the document]
Comment (thomaskrause) [reply docs/dev/src/SUMMARY.md]: Workflow needs to be re-organized (according to the FIXMEs) and I did not want to assume how the structure will looks like. If the re-organization PR includes this part into the workflow I'm fine with it, but currently I don't think we should mixup this information.

**Pull request merged**